Sources/ServiceLifecycle/Docs.docc/How to adopt ServiceLifecycle in applications.md

```diff
@@ -0,0 +1,200 @@
+# How to adopt ServiceLifecycle in applications
+
+``ServiceLifecycle`` aims to provide a unified API that services should adopt to make orchestrating
+them in an application easier. To achieve this ``ServiceLifecycle`` is providing the ``ServiceRunner`` actor.
+
+## Why do we need this?
+
+When building applications we often have a bunch of services that comprise the internals of the applications.
+These services include fundamental needs like logging or metrics. Moreover, they also include
+services that compromise the application's business logic such as long-running actors.
+Lastly, they might also include HTTP, gRPC, or similar servers that the application is exposing.
+One important requirement of the application is to orchestrate the various services currently during
+startup and shutdown. Furthermore, the application also needs to handle a single service failing.
+
+Swift introduced Structured Concurrency which already helps tremendously with running multiple 
+async services concurrently. This can be achieved with the use of task groups. However, Structured
+Concurrency doesn't enforce consistent interfaces between the services, so it becomes hard to orchestrate them.
+This is where ``ServiceLifecycle`` comes in. It provides the ``Service`` protocol which enforces 
+a common API. Additionally, it provides the ``ServiceRunner`` which is responsible for orchestrating
+all services in an application.
+
+## Adopting the ServiceRunner in your application
+
+This article is focusing on how the ``ServiceRunner`` works and how you can adopt it in your application.
+If you are interested in how to properly implement a service, go check out the article:  <doc:How-to-adopt-ServiceLifecycle-in-libraries>.
+
+### How is the ServiceRunner working?
+
+The ``ServiceRunner`` is just a slightly complicated task group under the hood that runs each service
```

Comment:
maybe we should call it ServiceGroup?

Reply:
I like the name `ServiceGroup` and renamed everything. Thanks!